charts/dependency-track/values.yaml

```diff
@@ -20,6 +20,8 @@ common:
     automount: false
 
 apiServer:
+# -- The type of deployment. Can be either deployment or statefulSet.
+  deploymentType: statefulSet
```

Comment:
Let's keep this in line with the "official" names and use `Deployment` and `StatefulSet` here?

Reply:
fixed in [4a83084](https://github.com/DependencyTrack/helm-charts/pull/102/commits/4a830849be358628d54a29f88abde1f2eb05148a)